challenge.js

```diff
@@ -25,7 +25,7 @@
  *	Answer below.
  */
 
-// ANSWER
+// ANSWER E could be a descendent of A, if we are supposed to assume that D comes after C and is a child of C.  But we don't have that information and can't assume that the person creating the stylesheet's understanding of how A, B, C, D, and E should be ordered.
```

Comment:
You are given that B, C, and D are siblings, which means they have the same parent. From this, you can deduce that E is a grandchild of A.